### PR TITLE
Added homework_37_220423 (CRUD + hibernate)

### PR DESCRIPTION
**Pull request by kate-lvlup**

### CODE REVIEW

**Comment by kate-lvlup** [comment]
Corrected the comments and moved everything to separate methods

File: homework_37_220423_hibernate/src/main/java/org/example/dao/MovieDao.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+public class MovieDao {
+
+    public void saveMovie(Movie movie) {
+        try (Session session = HibernateUtil.getSessionFactory().openSession()) {
+            Transaction transaction = session.beginTransaction();
+            session.save(movie);
+            transaction.commit();
+            session.close();
+            HibernateUtil.getSessionFactory().close();
```
Comment (YeaaaaH) [inline homework_37_220423_hibernate/src/main/java/org/example/dao/MovieDao.java:16]: do you really need to close it here?

File: homework_37_220423_hibernate/src/main/java/org/example/dao/MovieDao.java [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+import util.HibernateUtil;
+
+public class MovieDao {
+
+    public void saveMovie(Movie movie) {
+        try (Session session = HibernateUtil.getSessionFactory().openSession()) {
+            Transaction transaction = session.beginTransaction();
+            session.save(movie);
+            transaction.commit();
+            session.close();
```
Comment (YeaaaaH) [inline homework_37_220423_hibernate/src/main/java/org/example/dao/MovieDao.java:15]: you have try with resources

**Review by YeaaaaH — CHANGES_REQUESTED** [review]
fix it

File: homework_37_220423_hibernate/src/main/java/org/example/dao/MovieDao.java (hunk 1 above)
Comment (kate-lvlup) [reply homework_37_220423_hibernate/src/main/java/org/example/dao/MovieDao.java]: Created separated method closeSessionFactoryConnection()

File: homework_37_220423_hibernate/src/main/java/org/example/dao/MovieDao.java (hunk 2 above)
Comment (kate-lvlup) [reply homework_37_220423_hibernate/src/main/java/org/example/dao/MovieDao.java]: fix, session.close() row was deleted

**Comment by kate-lvlup** [comment]
all comments fixed

**Review by YeaaaaH — APPROVED** [review]

**Pull request merged**